pkg/validation/spec/swagger.go

```diff
@@ -123,6 +157,28 @@ func (s *SchemaOrBool) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+func (s *SchemaOrBool) UnmarshalNextJSON(opts jsonv2.UnmarshalOptions, dec *jsonv2.Decoder) error {
+	switch k := dec.PeekKind(); k {
+	case '{':
+		err := opts.UnmarshalNext(dec, &s.Schema)
+		if err != nil {
+			return err
+		}
+		s.Allows = true
+		return nil
+	case 't':
+		fallthrough
+	case 'f':
```

Comment:
I don't know if that's idiomatic, shouldn't you do `case 't', 'f':`?

Reply:
yup. changed